libs/talent-pool/config/src/lib/talent-pool-config.ts

```diff
@@ -745,3 +745,29 @@ export const howDidHearAboutRediOptions = {
   'social-media': 'Social Media',
   other: 'Other',
 } as const
+
+export const locationStates = {
+  'baden-wuerttemberg': 'Baden-Württemberg',
+  bayern: 'Bayern',
+  berlin: 'Berlin',
+  brandenburg: 'Brandenburg',
+  bremen: 'Bremen',
+  hamburg: 'Hamburg',
+  hessen: 'Hessen',
+  'mecklenburg-vorpommern': 'Mecklenburg-Vorpommern',
+  niedersachsen: 'Niedersachsen',
+  'nordrhein-westfalen': 'Nordrhein-Westfalen',
+  'rheinland-pfalz': 'Rheinland-Pfalz',
+  saarland: 'Saarland',
+  sachsen: 'Sachsen',
+  sachsenAnhalt: 'Sachsen-Anhalt',
+  schleswigHolstein: 'Schleswig-Holstein',
+  thueringen: 'Thüringen',
+} as const
+
+export const locationStatesOptions = Object.entries(locationStates).map(
```

Comment:
There's been a (turtle speed) refactoring going on in the CON and "common" codebase of moving the creation of these "computed properties" into the files that actually use them. It would lead to a bit of code duplication, but an entirely acceptable one since when we merge Pablo's PR we'll be using some nice helper functions he created, that cover the ...
`Object.entries(options).map(([value, label]) => ({ value, label }))`
... part.

For this reason, can we move this block into the file that actually uses the properties?

Reply:
Also, please consider the variable naming - IMHO we should have `germany` in it